### PR TITLE
[WTEL-5558] feature: Ability to send files/images in mailings

### PR DESCRIPTION
**Pull request by nazarwebitel**
# Pull Request

## Jira

[WTEL-5558](https://webitel.atlassian.net/browse/WTEL-5558) - Ability to send files/images in mailings

## Type of Change
- [x] Feature
- [ ] Bugfix
- [ ] Refactor
- [ ] Documentation
- [ ] Other (please describe): [Insert description]

[WTEL-5558]: https://webitel.atlassian.net/browse/WTEL-5558?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: bot/telegram/internal/builder/send_message.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+		document.Caption = caption
+		b.messageConfig = &document
+		b.baseChat = &document.BaseChat
+	}
+
+	return nil
+}
+
+// IsKeyboadSetted returns bool true if keyboard is setted
+func (b *SendMessageBuilder) IsKeyboadSetted() bool {
```
Comment (nazarwebitel) [inline bot/telegram/internal/builder/send_message.go:107]: Incorrect function name

File: proto/storage/file.proto [hunk 2]
```diff
@@ -4,11 +4,18 @@ option go_package = "github.com/webitel/chat_manager/api/proto/storage";
...
 
-// import "google/api/annotations.proto";
+import "google/api/annotations.proto";
 
 service FileService {
     rpc UploadFile (stream UploadFileRequest) returns (UploadFileResponse) {}
     rpc UploadFileUrl (UploadFileUrlRequest) returns (UploadFileUrlResponse) {}
+    rpc GenerateFileLink (GenerateFileLinkRequest) returns (GenerateFileLinkResponse) {}
+
+    rpc SearchFiles(SearchFilesRequest) returns (ListFile) {
```
Comment (nazarwebitel) [inline proto/storage/file.proto:14]: Delete this rpc endpoint

File: bot/service.go [hunk 3]
```diff
@@ -112,7 +118,7 @@ func (srv *Service) onStart() {
 		// Term: "",
 		Filter: map[string]interface{}{
 			// LoadAndStart providers which must init their client connections
-			"provider": []string{"gotd"},
+			"provider": []string{"gotd11"},
```
Comment (nazarwebitel) [inline bot/service.go:121]: Cancel this change, it should stay local

**Comment by srgdemon** [comment]
Branch MERGED with changes ...

**Pull request closed without merging**